pupil-api/yarn.lock

```diff
@@ -17,7 +17,7 @@
   dependencies:
     tslib "^2.2.0"
 
-"@azure/core-amqp@^3.3.0":
+"@azure/core-amqp@^3.1.0":
```

Comment:
Why the downgrade here as well as in PR #2511 ?  Any security issue for us?

Reply:
not sure why this is in here either.  will roll it back